server/trillian_log_server/main.go

```diff
@@ -59,6 +59,7 @@ func startRPCServer(registry extension.Registry) (*grpc.Server, error) {
 	return grpcServer, nil
 }
 
+// TODO(drysdale): commonize the following 2 fns across all main()s
```

Comment:
Look out for conflicts with [PR 418](https://github.com/google/trillian/pull/418/commits/575bac40806fac13fffc86cd0b0cf12b6185d01e) if you go for startHTTPServer too.